Frameworks/UIKit/UILabel.mm

```diff
@@ -129,11 +132,24 @@ - (void)_adjustFontSizeToFit {
 
     // Special:on reference platform, adjustFontSizeToFit is no-op when lineBreakMode is Wrapping
     if (_lineBreakMode != UILineBreakModeWordWrap && _lineBreakMode != UILineBreakModeCharacterWrap) {
+        float minimumFontSize = _minimumFontSize;
+
+        // if minimumScaleFactor is used, it should override _minimumFontSize which is deprecated
```

Comment:
yes. they are different and separate and independent of each other.

---
In reply to: [99672980](https://github.com/Microsoft/WinObjC/pull/1918#discussion_r99672980) [](ancestors = 99672980)